auto_release.sh

```diff
@@ -0,0 +1,118 @@
+#! /bin/sh
+
+###### Script writer: Shivansh Saini (@shivanshs9) [26/10/2018] ######
+
+## Initial Configurations
+REPO_URI="sendgrid/python-http-client"
+GITHUB_USERNAME="sendgrid"
+CHANGELOG_FILE="CHANGELOG.md"
+
+## Helper functions
+get_line_no() {
+	release="$1"
+	grep -n -m 1 "$release" "$CHANGELOG_FILE" | cut -d ":" -f 1
+}
+
+get_current_tag() {
+	git describe --abbrev=0 --tags
+}
+
+get_latest_tag() {
+	curl -s $API_RELEASES_URL"/latest" | python3 -c \
+		"import sys, json; print(json.load(sys.stdin)['tag_name'])" 2>/dev/null
```

Comment:
Because this is a bash script, it might be cleaner to use the `jq` tool here instead of inlining python.

Reply:
I know, but I wasn't sure if it's fine to include an additional dependency in this PR. Python will definitely be installed in the development system, since this is a python client but **jq** may not be available.